vaadin-messages-flow-parent/vaadin-messages-flow-integration-tests/src/test/java/com/vaadin/flow/component/messages/tests/MessageListIT.java

```diff
@@ -153,7 +168,8 @@ public void setImageAsStreamResource_imageLoaded() {
         clickElementWithJs("setImageAsStreamResource");
         String imageUrl = messageList.getMessageElements().get(0).getUserImg();
         MatcherAssert.assertThat(imageUrl, startsWith("VAADIN/dynamic"));
-        checkLogsForErrors(); // would fail if the image wasn't hosted
+        // would fail if the avatar.png image wasn't hosted
+        checkLogsForErrors(message -> message.contains("test.jpg"));
```

Comment:
This test was also failing on `main` (at least locally). Needed to update it to get a green build.